specification.markdown

```diff
@@ -427,17 +417,30 @@ The id attribute is REQUIRED.
 
 Within an EBML Schema, the XPath of `@minOccurs` attribute is `/EBMLSchema/element/@minOccurs`.
 
-An integer expressing the minimum permitted number of occurrences of this EBML Element within its Parent Element. The minOccurs value MUST be equal to the EBMLMinOccurrence value of the path.
+The minOccurs is an integer representing the minimum permitted number of occurrences of this EBML Element within its Parent Element.
+
+Each instance of the Parent Element MUST contain at least this many instances of this EBML Element.
```

Comment:
Perhaps 'contain' is the wrong word. As minOccurs=1 with a default value does not need to be 'contained' within the parent.

Reply:
This is the same wording that was used in `EBMLMinOccurrence`, dating back to 40fe4cb33d7e5d06e9f58d389f95d425437071ba.

In the next sentence(s) it actually explains in which case the element can be omitted: `EBML Elements with minOccurs set to "1" that also have a default value declared are not REQUIRED to be stored`.
It was all on the same line before and still in the same paragraph.

For maxOccurs there's no such thing. How about: 
`Each instance of the Parent Element MUST contain at most this many instances of this EBML Element, including the unwritten mandatory element with a default value, see (#note-on-the-use-of-default-attributes-to-define-mandatory-ebml-elements).`